src/bin/init/setupVitePluginIfNeeded.ts

```
@@ -0,0 +1,143 @@
+
+                    pluginsProp.value.elements.push(keycloakifyCall);
+                }
+            }
+
+            this.traverse(path);
+        }
+    });
+
+    let viteConfigTsContent_modified = babelGenerate(root).code;
+
+    format: {
```

Biome (1.9.4)

[error] Unexpected label.

Only loops should be labeled.
The use of labels for other statements is suspicious and unfamiliar.

(lint/suspicious/noConfusingLabels)